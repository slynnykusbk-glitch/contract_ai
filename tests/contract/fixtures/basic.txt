This Agreement is made between Alpha Corp and Beta Ltd. The parties agree to keep all information confidential and comply with applicable law.
