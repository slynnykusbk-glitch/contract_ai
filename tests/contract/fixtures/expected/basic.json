{
  "analysis": {
    "findings": [],
    "status": "ok"
  },
  "clauses": [],
  "document": {},
  "findings": [],
  "meta": {
    "active_packs": [],
    "document_type": "unknown",
    "fired_rules": [],
    "language": "en-GB",
    "model": "mock-static",
    "provider": "mock",
    "rules_evaluated": 0,
    "rules_fired_count": 0,
    "rules_loaded_count": 0,
    "text_bytes": 142,
    "valid_config": true
  },
  "recommendations": [],
  "results": {
    "summary": {
      "carveouts": {
        "carveouts": [
          "confidentiality"
        ],
        "has_carveouts": true,
        "list": [
          "confidentiality"
        ]
      },
      "clause_type": "confidentiality",
      "conditions_vs_warranties": {
        "explicit_conditions": [],
        "explicit_warranties": [],
        "has_conditions": false,
        "has_warranties": false
      },
      "currency": null,
      "dates": {
        "commencement": null,
        "dated": null,
        "effective": null
      },
      "debug": {
        "doctype_top": [
          {
            "score": 0.0,
            "type": "NDA"
          },
          {
            "score": 0.0,
            "type": "MSA (Services)"
          },
          {
            "score": 0.0,
            "type": "Supply of Goods"
          },
          {
            "score": 0.0,
            "type": "DPA"
          },
          {
            "score": 0.0,
            "type": "License (IP)"
          }
        ]
      },
      "doc_type": {
        "candidates": [
          {
            "score": 0.0,
            "type": "unknown"
          }
        ],
        "confidence": 0.0,
        "top": {
          "score": 0.0,
          "type": "unknown"
        }
      },
      "exclusivity": null,
      "governing_law": null,
      "hints": [
        "confidentiality"
      ],
      "jurisdiction": null,
      "liability": {
        "cap_currency": null,
        "cap_value": null,
        "has_cap": false,
        "notes": null
      },
      "parties": [
        {
          "name": "Alpha Corp"
        },
        {
          "name": "Beta Ltd"
        }
      ],
      "rules_count": 176,
      "signatures": [],
      "term": {
        "end": null,
        "mode": "unknown",
        "renew_notice": null,
        "start": null
      },
      "type": "unknown",
      "type_confidence": 0.0,
      "type_source": null
    }
  },
  "rules_coverage": {
    "doc_type": {
      "source": null,
      "value": "unknown"
    },
    "rules": []
  },
  "schema_version": "1.4",
  "status": "ok",
  "summary": {
    "carveouts": {
      "carveouts": [
        "confidentiality"
      ],
      "has_carveouts": true,
      "list": [
        "confidentiality"
      ]
    },
    "clause_type": "confidentiality",
    "conditions_vs_warranties": {
      "explicit_conditions": [],
      "explicit_warranties": [],
      "has_conditions": false,
      "has_warranties": false
    },
    "currency": null,
    "dates": {
      "commencement": null,
      "dated": null,
      "effective": null
    },
    "debug": {
      "doctype_top": [
        {
          "score": 0.0,
          "type": "NDA"
        },
        {
          "score": 0.0,
          "type": "MSA (Services)"
        },
        {
          "score": 0.0,
          "type": "Supply of Goods"
        },
        {
          "score": 0.0,
          "type": "DPA"
        },
        {
          "score": 0.0,
          "type": "License (IP)"
        }
      ]
    },
    "doc_type": {
      "candidates": [
        {
          "score": 0.0,
          "type": "unknown"
        }
      ],
      "confidence": 0.0,
      "top": {
        "score": 0.0,
        "type": "unknown"
      }
    },
    "exclusivity": null,
    "governing_law": null,
    "hints": [
      "confidentiality"
    ],
    "jurisdiction": null,
    "liability": {
      "cap_currency": null,
      "cap_value": null,
      "has_cap": false,
      "notes": null
    },
    "parties": [
      {
        "name": "Alpha Corp"
      },
      {
        "name": "Beta Ltd"
      }
    ],
    "rules_count": 176,
    "signatures": [],
    "term": {
      "end": null,
      "mode": "unknown",
      "renew_notice": null,
      "start": null
    },
    "type": "unknown",
    "type_confidence": 0.0,
    "type_source": null
  }
}